src/main/resources/application.yml

```diff
@@ -123,3 +131,6 @@ log-client-credentials-jwt-info: false
 upstream-timeout-ms: 10000
 case-notes-service-upstream-timeout-ms: 30000
 max-response-in-memory-size-bytes: 750000
+
+preemptive-cache-key-prefix: ""
```

Comment:
Does this need a value?

Reply:
yes, needed as part of this change. 
I think we plan on adding the prefix for each of the caches we have.